snap/snapcraft.yaml

```diff
@@ -0,0 +1,23 @@
+name: bigchaindb
+version: master
```

Comment:
we should probably match the current version that is in [version.py](https://github.com/bigchaindb/bigchaindb/blob/master/bigchaindb/version.py): `0.11.dev`